beetmoverscript/constants.py

```diff
@@ -11,13 +13,17 @@
     '.xpi': 'application/zip',
     '.apk': 'application/vnd.android.package-archive',
 }
-
-PLATFORM_MAP = {
+STAGE_PLATFORM_MAP = {
     'linux': 'linux-i686',
     'linux64': 'linux-x86_64',
 }
-
+# TODO don't rely on default dict
```

Comment:
sort of. This code copies this logic:
https://github.com/mozilla-releng/beetmoverscript/blob/47aa586ad10632a178991a01dad8e1335ae2b017/beetmoverscript/utils.py#L58-L62

I could put something like `'linux': 'firefox'`  lines in here. But that would require me to sanity check what the linux 32 and 64 names would be and all the other osx/win platforms. So I was sort of lazy here. :)

I figured we could be more explicit when we port osx/win over.